mlrun/api/main.py

```diff
@@ -210,6 +220,70 @@ def _start_periodic_runs_monitoring():
         )
 
 
+def _start_periodic_pulling_of_clusterization_spec():
+    interval = int(config.httpdb.clusterization.worker.sync_with_chief.interval)
+    if interval > 0:
+        logger.info("Starting periodic chief state pulling", interval=interval)
```

Comment:
1. Log doesn't match function name
2. IMO both the log and the function name are not good, the purpose is to **sync the worker state to the chief state** (and stop when it's terminal since then it can't change), so maybe `_start_chief_state_sync_loop` and `"Starting chief state sync loop`

Reply:
The main Idea was that maybe in the future we will want to use that pulling mechanism for other stuff rather than chief state. That is why I didn't call any of the triggering functions related to chief state but rather clusterization spec and then inside of it we will implement the synchronization logic that we want to apply from the chief.